### PR TITLE
refactor(demo-integrations): rewriting e2e tests to CT (tests for native `maxlength` attribute)

### PR DESCRIPTION
**Pull request by KrollikRoddzer**
Closes a part of https://github.com/taiga-family/maskito/issues/886
Done:
1. Rewritten all tests from e2e tests to component tests for native max length
2. Deleted all tests
3. Added support of 'cypress-real-events' for component testing
4. Added field maxLength for TestInput

### CODE REVIEW

File: projects/demo-integrations/src/tests/component-testing/native-max-length/native-maxlength-attribute.cy.ts [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
-            cy.visit(DemoPath.Cypress);
-            cy.get('#maxlength input[maxlength="3"]')
+            const maskitoOptions = maskitoNumberOptionsGenerator({
+                thousandSeparator: ' ',
+            });
+
+            cy.mount(TestInput, {
+                componentProperties: {
+                    maskitoOptions,
+                    maxLength: '3',
```
Comment (nsbarsukov) [inline projects/demo-integrations/src/tests/component-testing/native-max-length/native-maxlength-attribute.cy.ts:17]: Why `string` ?

Let's use `number`

File: projects/demo-integrations/src/tests/component-testing/utils.ts [hunk 2]
```diff
@@ -22,4 +34,13 @@ export class TestInput {
 
     @Output()
     input = new EventEmitter();
+
+    @Input()
+    maxLength?: string;
+
+    ngAfterViewInit(): void {
+        if (this.maxLength) {
+            this.inputElement.nativeElement.maxLength = this.maxLength;
```
Comment (nsbarsukov) [inline projects/demo-integrations/src/tests/component-testing/utils.ts:43]: Why so ?

Read this section of documentation:
https://angular.dev/guide/templates/attribute-binding#syntax

**Review by nsbarsukov — CHANGES_REQUESTED** [review]
Rename PR's title according to conventional commit specification:

https://www.conventionalcommits.org/en/v1.0.0/

File: projects/demo-integrations/src/tests/component-testing/utils.ts (hunk 2 above)
Comment (KrollikRoddzer) [reply projects/demo-integrations/src/tests/component-testing/utils.ts]: Done

File: projects/demo-integrations/src/tests/component-testing/native-max-length/native-maxlength-attribute.cy.ts (hunk 1 above)
Comment (KrollikRoddzer) [reply projects/demo-integrations/src/tests/component-testing/native-max-length/native-maxlength-attribute.cy.ts]: Done

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo-integrations/src/tests/component-testing/utils.ts [hunk 3]
```diff
@@ -22,4 +23,7 @@ export class TestInput {
 
     @Output()
     input = new EventEmitter();
+
+    @Input()
+    maxLength: number = Infinity;
```
Comment (nsbarsukov) [inline projects/demo-integrations/src/tests/component-testing/utils.ts:28]: Fix [eslint errors](https://github.com/taiga-family/maskito/actions/runs/8054597253/job/21999511308?pr=1084), please

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**